privaterelay/views.py

```diff
@@ -222,11 +260,12 @@ def _handle_fxa_profile_change(authentic_jwt, social_account, event_key):
         sentry_sdk.capture_exception(e)
         return HttpResponse('202 Accepted', status=202)
 
-    logger.info('fxa_rp_event', extra={
-        'fxa_uid': authentic_jwt['sub'],
-        'event_key': event_key,
-        'real_address': sha256(new_email.encode('utf-8')).hexdigest(),
-    })
+    if authentic_jwt and event_key:
```

Comment:
I suggest we check that`social_account` and `authentic_jwt` is not None  earlier in the function and throw and error if it is None. Technically, the `authentic_jwt` should never be None right? If `authentic_jwt` was None we should not edit the profile changes since we cannot verify that the update is truly from FXA.

Reply:
Hmm ... I added the ability to pass `None` so I could force a refresh of the FXA profile data by passing `fxa_refresh=1` in the URL. (So when the user gets back to Relay after purchasing a premium subscription, we could force a profile refresh to see the subscription in their updated fxa profile data.)

But yeah - this introduces a bypass which doesn't feel right. There should really be a `_refresh_fxa_profile()` function apart from `_handle_fxa_profile_change`. Lemme work on that as a refactor.

Reply:
Separate issue created here https://github.com/mozilla/fx-private-relay/issues/809